nusantara/nusa_datasets/id_short_answer_grading/utils/id_short_answer_grading_utils.py

```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+from pathlib import Path
+
+def createSaintekAndSoshumDataset(saintek_question: Path, soshum_question: Path, saintek_score: Path, soshum_score: Path):
+    # creating dataframes
+    saintek_question = pd.read_csv(saintek_question, sep=",", header="infer")
+    saintek_score = pd.read_csv(saintek_score, sep=",", header="infer")
+    soshum_question = pd.read_csv(soshum_question, sep=",", header="infer")
+    soshum_score = pd.read_csv(soshum_score, sep=",", header="infer")
+
+    # renaming
+    saintek_question.rename(columns={'jawaban': 'kunci-jawaban'}, inplace='true')
+    soshum_question.rename(columns={'jawaban': 'kunci-jawaban'}, inplace='true')
+
+    # joining
+    saintek_joined = pd.merge(saintek_question, saintek_score, how='left', on = 'type-problem')
+    soshum_joined = pd.merge(soshum_question, soshum_score, how='left', on = 'type-problem')
+
+    # adding answer key
+    saintek_joined = saintek_joined.append(answerKeyScoreDataset(saintek_question), ignore_index=True)
+    soshum_joined = soshum_joined.append(answerKeyScoreDataset(soshum_question), ignore_index=True)
+
+    # re-case-ing (kebab to snake)
+    saintek_joined.rename(columns={'type-problem': 'type_problem', 'kunci-jawaban': 'kunci_jawaban'}, inplace=True)
+    soshum_joined.rename(columns={'type-problem': 'type_problem', 'kunci-jawaban': 'kunci_jawaban'}, inplace=True)
+
+    # concatenate
+    saintek_and_soshum = pd.concat([saintek_joined, soshum_joined], ignore_index=True).reset_index()
+
+    return saintek_and_soshum
+
+
+def answerKeyScoreDataset(question: pd.DataFrame):
```

Comment:
nit: can we change the function name to be more descriptive? e.g `build_answer_key_score_dataframe`